src/main/java/world/bentobox/bentobox/schems/Clipboard.java

```diff
@@ -257,14 +296,10 @@ private void pasteBlock(World world, Island island, Location location, Configura
         Block block = world.getBlockAt(x, y, z);
         String blockData = config.getString("bd");
         if (blockData != null) {
-            if (config.getBoolean(ATTACHED)) {
-                plugin.getServer().getScheduler().runTask(plugin, () -> setBlock(island, block, config, blockData));
-            } else {
-                setBlock(island, block, config, blockData);
-            }
+            setBlock(island, block, config, blockData);
         }
-        // Entities
-        if (config.isConfigurationSection(ENTITY)) {
+        // Entities - only paste here if this is a clipboard paste, otherwise, wait until island is pasted
+        if (pasteEntities && config.isConfigurationSection(ENTITY)) {
```

Comment:
Pasting using the clipboard is NOT done async right now, just when pasting an island in the game. So using the schem paste command may lag the server. This will need changing in the future once the async pasting works completely.

Reply:
Now it's done async too.